src/postgraphql/cli.ts

```diff
@@ -122,6 +123,7 @@ const server = createServer(postgraphql(pgConfig, schemas, {
   pgDefaultRole,
   watchPg,
   showErrorStack,
+  extendedErrors,
```

Comment:
You've added the options here, but you've not added them to the list on lines 23-45? Also you may need to coerce to array here by splitting on commas?